tracing02-xdp-monitor/Makefile

```diff
@@ -4,10 +4,10 @@
 XDP_TARGETS  := trace_prog_kern
 USER_TARGETS := trace_load_and_stats
 
-COMMON_DIR = ../common
+COMMON_DIR := ../common
 
 # Extend with another COMMON_OBJS
-COMMON_OBJS += $(COMMON_DIR)/common_user_bpf_xdp.o
+COMMON_OBJS := $(COMMON_DIR)/common_user_bpf_xdp.o
```

Comment:
This looks wrong, comment says this is extending the variable

Reply:
Thank you for your feedback. This line is clearly an initial assignment, so := is the correct operator to use here. It seems the comment might be misleading.

Reply:
Yup, definitely the comment that is wrong :)